provider/data_source_instance_test.go

```diff
@@ -0,0 +1,88 @@
+package provider
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/bytebase/terraform-provider-bytebase/provider/internal"
+)
+
+func TestAccInstanceDataSource(t *testing.T) {
+	identifier := "new_instance"
```

Comment:
what does Acc stand for?

Reply:
The `Acc` stands for `Acceptance`. According to the docs,

> To create an acceptance test in the example resource_example_compute_test.go file, the function name must begin with TestAccXxx, and have the following signature:
> func TestAccXxx(*testing.T)